src/mlpack/core/optimizers/scd/descent_policies/random_descent.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file random_descent.hpp
+ * @author Shikhar Bhardwaj
+ *
+ * Random descent policy for Stochastic Co ordinate Descent (SCD).
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SCD_DESCENT_POLICIES_RANDOM_HPP
+#define MLPACK_CORE_OPTIMIZERS_SCD_DESCENT_POLICIES_RANDOM_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Random descent policy for Stochastic Co-ordinate Descent(SCD). This
+ * descent scheme picks a the co-ordinate for the descent uniformly randomly.
+ */
+class RandomDescent
+{
+ public:
+  /**
+   * The DescentFeature method is used to get the descent coordinate for the
+   * current iteration.
+   *
+   * @tparam ResolvableFunctionType The type of the function to be optimized.
```

Comment:
Should we have a pointer to the more detailed description of what the signature of this function is in `scd.hpp`? Something like "For more information see the SCD implementation"